### PR TITLE
remove stabilized feature

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

**Comment by RalfJung** [comment]
@oli-obk 	this required a fix in miri-rustc-tests, could you have a look?

File: src/bin/miri-rustc-tests.rs [hunk 1]
```diff
@@ -47,7 +47,7 @@ impl rustc_driver::Callbacks for MiriCompilerCalls {
                 impl<'a, 'tcx: 'a, 'hir> itemlikevisit::ItemLikeVisitor<'hir> for Visitor<'a, 'tcx> {
                     fn visit_item(&mut self, i: &'hir hir::Item) {
                         if let hir::ItemKind::Fn(.., body_id) = i.node {
-                            if i.attrs.iter().any(|attr| attr.name() == "test") {
+                            if i.attrs.iter().any(|attr| attr.ident_str() == Some("test")) {
```
Comment (oli-obk) [inline src/bin/miri-rustc-tests.rs]:
```suggestion
                            if i.attrs.iter().any(|attr| attr.check_name("test")) {
```

Comment (RalfJung) [reply src/bin/miri-rustc-tests.rs]: that "marks the attribute as used", is that really what we want?

Comment (oli-obk) [reply src/bin/miri-rustc-tests.rs]: I can't think of a way it would hurt.

**Pull request merged**